meta-genivi-dev/recipes-multimedia/alsa/alsa-state.bbappend

```diff
@@ -1 +1,9 @@
 FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+
+SRC_URI_append_m3ulcb = "\
+  file://m3ulcb.state \
+"
+
+do_install_append_m3ulcb() {
+  mv ${D}${localstatedir}/lib/alsa/m3ulcb.state ${D}${localstatedir}/lib/alsa/asound.state
```

Comment:
I was using install command but due to some weird reasons, I will end up with m3ulcb.state and asound.state file installed. There is some magic involved that I'm avoiding with mv here.

This is a hack and we need to figure this out after the release but if this is working for all targets, we need to push this so we can rollout 12.0 today.